source/learn/rosetta_stone.md

```diff
@@ -0,0 +1,1513 @@
+# Python Fortran Rosetta Stone
+
+Python with NumPy and Fortran are very similar in terms of
+expressiveness and features. This rosetta stone shows how to implement
+many common idioms in both languages side by side.
+
+## How to Execute Code Snippets
+
+Consider for example the following code snippets:
+
+<table>
+<colgroup>
+<col style="width: 49%" />
+<col style="width: 50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td>NumPy</td>
+<td><blockquote>
+<p>Fortran</p>
+</blockquote></td>
```

Comment:
That's rather cumbersome to edit, can we find a pure MyST equivalent? Maybe something like

```````md
:::::{grid}
::::{grid-item}
```{code-block} python
:caption: Python
# ...
```
::::
::::{grid-item}
```{code-block} fortran
:caption: Fortran
! ...
```
::::
:::::
```````

I have to admit that MyST seems to be only a minor improvement over raw HTML in this context.

Reply:
Sure sir. Thanks